esp_c3_m1_i_kit/pins_arduino.h

```diff
@@ -0,0 +1,40 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define EXTERNAL_NUM_INTERRUPTS 30
+#define NUM_DIGITAL_PINS        30
+#define NUM_ANALOG_INPUTS       6
+
+#define analogInputToDigitalPin(p)  (((p)<NUM_ANALOG_INPUTS)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<NUM_DIGITAL_PINS)?(p):-1)
+#define digitalPinHasPWM(p)         (p < EXTERNAL_NUM_INTERRUPTS)
+
+static const uint8_t LED_BUILTIN = 18;
+static const uint8_t LED_BUILTIN = 19;
+#define BUILTIN_LED  LED_BUILTIN // backward compatibility
+
+static const uint8_t LED_GREEN = 4;
+static const uint8_t LED_RED = 3;
+static const uint8_t LED_BLUE = 5;
+
+static const uint8_t TX = 18;
+static const uint8_t RX = 19;
```

Comment:
This is the correct pins used in this boards. GPIO18 and 19 are LEDs and USB JTAG/CDC.

```
static const uint8_t TX = 21;
static const uint8_t RX = 20;
```

Reply:
Fixed